xpk.py

```diff
@@ -3187,18 +3224,25 @@ def directory_path_type(value):
 
 ### Required arguments
 workload_delete_parser_required_arguments.add_argument(
+    '--cluster',
+    type=str,
+    default=None,
+    help='The name of the cluster to delete the job on.',
+    required=True,
+)
+
+### Optional arguments
+workload_delete_parser_optional_arguments.add_argument(
     '--workload',
     type=workload_name_type,
     default=None,
     help='The name of the workload to delete.',
```

Comment:
Can you add that if the workload is not specified, the default behavior is that all workloads will be deleted from the cluster?

Reply:
Added!